### ISSUE TITLE
P1 intégrer le système de combo

### ISSUE THREAD

**Issue by LucaOGameDesign** [nanaki13/rpg#1]
Page 7:
https://docs.google.com/document/d/1KhLrcJlDgGTtKfrwDq1tMTYVyFY-rYPae58vl4knb-E/edit#

**Comment by nanaki13** [comment]
Ces combos visent à faire plus de dégâts au détriment d’un certain nombre d’affect: En effet, seuls les affects d’un seul des participants prendront effet lors de l’attaque. Le bonus de dégât est non négligeable… attention cependant un personnage attaqué alors qu’il est en posture de combo (attente de partenaire) prendra plus de dégâts (X2) et se fera CANCEL.

Calculs dégâts combo: (n= le nombre de participants au combo)

(dégâts attaquant1 + dégâts attaquant2  sans les AFFECTS)) * (n+1)


NOTE: Anim de combo ala persona5 (avec png des persos en plus grand et parallax) (ou mieux si bcp d’argent)

**Comment by nanaki13** [comment]
@LucaOGameDesign  : Combo est une commande a ajouter au perso alors?

**Comment by LucaOGameDesign** [comment]
Yes @nanaki13  ! Pour le moment c'est designé comme une commande. Quand tu la sélectionnes il faut ensuite choisir le ou les partenaire avec qui tu vas combo, choisir "OK" une fois que tous les participants souhaités on été sélectionnés, puis l'arme utilisée (QTE) et enfin la cible visée. ça met l'utilisateur en pause sur "COM" jusqu'à ce que le ou tous les autres partenaires choisis atteignent lui/eux aussi "COM".
Une fois réunis sur "COM", les participants reprennent ensuite leur route vers la portion "ACT", à la vitesse de l'initiateur du combo pour enfin réaliser l'attaque.
